Venmo/src/main/java/com/braintreepayments/api/VenmoRequest.java

```diff
@@ -248,6 +249,22 @@ public ArrayList<VenmoLineItem> getLineItems() {
         return lineItems;
     }
 
+    /**
+     * @param finalAmount Optional - Indicates whether the purchase amount is the final amount.
+     *                    Defaults to false.
+     */
+    public void setFinalAmount(boolean finalAmount) {
```

Comment:
Nitpick - I feel like `setIsFinalAmount` would make it more clear that we are setting a boolean here, not a final amount dollar value - what do you think?

Also, I let's add a public getter for this too for consistency and Kotlin property interop

Reply:
That totally makes sense, I went back and fourth on it. Updated here: https://github.com/braintree/braintree_android/pull/921/commits/cbdecdc6a88094b13ed7543c474ec628ef68068f